.github/workflows/rake.yml

```diff
@@ -27,7 +25,7 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
 
-      - uses: metanorma/metanorma-build-scripts/plantuml-setup-action@master
+      - uses: metanorma/metanorma-build-scripts/plantuml-setup-action@main
```

Comment:
@CAMOBAP why do we need PlantUML installed?

Reply:
Good question we always had it here, let me check if it will works without plantuml